src/NUnitFramework/framework/Internal/ParameterWrapper.cs

```diff
@@ -24,8 +24,13 @@
 using System;
 using System.Collections.Generic;
 using System.Reflection;
+using NUnit.Framework.Compatibility;
 using NUnit.Framework.Interfaces;
 
+#if NETCORE
+using System.Linq;
```

Comment:
I see it saves a line or two of code to use Linq, but it adds another #if, which seems like a poor tradeoff. I noticed this in a few other places as well.